Code/Editor/EditorFramework/EditorApp/Startup.cpp

```diff
@@ -414,3 +417,26 @@ void ezQtEditorApp::CreatePanels()
 
   pAssetBrowserPanel->raise();
 }
+
+
+void ezQtEditorApp::SetupAndShowSplashScreen()
+{
+  EZ_ASSERT_DEV(m_pSplashScreen == nullptr, "Splash screen shouldn't exist already.");
+
+  QPixmap splashPixmap(":/Splash/Splash/splash.png");
```

Comment:
I assume we can't use the SVG directly?

Reply:
https://stackoverflow.com/questions/10079011/qpixmap-and-svg

Reply:
The problem is that we're on Qt 5.13 - that doesn't have the capability to keep the aspect ratio during SVG rendering. I'd need to hard code the aspect ratio into the code loading the SVG so that the image is not stretched. Not an ideal solution :(

Reply:
I switched to a the SVG now and added a TODO to go from the awkward size to a square size and using the aspect ratio mode of the SVG renderer when we migrate to a newer Qt verison.